src/analysis/processing/qgsalgorithmconcavehullbyfeature.cpp

```diff
@@ -0,0 +1,146 @@
+/***************************************************************************
+                         qgsalgorithmconcavehullbyfeature.cpp
+                         ---------------------
+    begin                : May 2025
+    copyright            : (C) 2025 by Nyall Dawson
+    email                : nyall dot dawson at gmail dot com
+ ***************************************************************************/
+
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#include "qgsalgorithmconcavehullbyfeature.h"
+#include "qgsgeometrycollection.h"
+
+///@cond PRIVATE
+
+QString QgsConcaveHullByFeatureAlgorithm::name() const
+{
+  return QStringLiteral( "concavehullbyfeature" );
+}
+
+QString QgsConcaveHullByFeatureAlgorithm::displayName() const
+{
+  return QObject::tr( "Concave hull (by feature)" );
+}
+
+QStringList QgsConcaveHullByFeatureAlgorithm::tags() const
+{
+  return QObject::tr( "concave,hull,bounds,bounding" ).split( ',' );
```

Comment:
Can we add 
```suggestion
  return QObject::tr( "concave,hull,bounds,bounding,convex" ).split( ',' );
```
Convenient when you remember convex and not concave, and technically you can create convex hull.